apps/bfd-data-fda/Jenkinsfile

```
@@ -0,0 +1,138 @@
+#!/usr/bin/env groovy
+
```

A brief, high-level overview of what this Jenkinsfile is for would be helpful, e.g.
```suggestion
/** 
 * Enables Conditional daily build of a latest bfd-data-fda SNAPSHOT artifact from master using the bfd-cbc-build container image.
 * Delivers to bfd-mgmt AWS CodeArtifact Maven Repository as appropriate.
 */
```